### ISSUE TITLE
Compilation failure in native IntelliJ import

### ISSUE THREAD

**Issue by ljacomet** [gradle/gradle#8003]
When using IntelliJ native Gradle import, sometimes the following happens:

* Compilation fails in buildSrc/subprojects/kotlin-dsl/src/main/kotlin/accessors/accessors.kt
* The IJ compiler contains the following warnings:
```
Warning:Kotlin: Classpath entry points to a non-existent location: .../build/external/files/gradle-kotlin-dsl-1.0-rc-12.jar
Warning:Kotlin: Classpath entry points to a non-existent location: .../build/external/files/gradle-kotlin-dsl-tooling-models-1.0-rc-12.jar
Warning:Kotlin: Classpath entry points to a non-existent location: .../build/external/files/kotlin-reflect-1.3.0-rc-116.jar
Warning:Kotlin: Classpath entry points to a non-existent location: .../build/external/files/kotlin-stdlib-1.3.0-rc-116.jar
Warning:Kotlin: Classpath entry points to a non-existent location: .../build/external/files/kotlin-stdlib-common-1.3.0-rc-116.jar
Warning:Kotlin: Classpath entry points to a non-existent location: .../build/external/files/kotlin-stdlib-jdk7-1.3.0-rc-116.jar
Warning:Kotlin: Classpath entry points to a non-existent location: .../build/external/files/kotlin-stdlib-jdk8-1.3.0-rc-116.jar
```

This is most likely caused by an update of the Kotlin DSL version and thus an update in the names of the generated files in `build/external/files`.
The main pain point is that if you are working on two different branches with different Kotlin DSL version, you have to apply the solution below after each branch switch.

Solution:
* Re-import the project from scratch. That is close the current project, import it again, saying yes to overwrite current configuration.

**Comment by wolfs** [comment]
@eskatos IIRC, the problem was that intellij was somehow using the files produced by `patchExternalModules` in `buildSrc`, right? Is this a bug in Gradle or in IDEA?

**Comment by wolfs** [comment]
Is there an issue on the Jetbrains Youtrack?

**Comment by mockitoguy** [comment]
I'm seeing the same problem. My workaround is to use "./gradlew idea" and open the *.ipr file in IDEA. That works and let's me compile / run tests in IDEA.

Consider fixing or updating the [CONTRIBUTING.md](https://github.com/gradle/gradle/blob/master/CONTRIBUTING.md#development-setup) because it advertises to use native import.

**Comment by eskatos** [comment]
This should be fixed now on master by #8050, I couldn't reproduce.
Please reopen if it happens again
